packages/navigator/src/browser/navigator-contribution.ts

```diff
@@ -357,6 +367,40 @@ export class FileNavigatorContribution extends AbstractViewContribution<FileNavi
                 });
             }
         });
+        registry.registerCommand(OpenEditorsCommands.CLOSE_ALL_TABS_FROM_TOOLBAR, {
+            execute: widget => this.withOpenEditorsWidget(widget, () => this.editorWidgets.forEach(editor => editor.close())),
+            isEnabled: widget => this.withOpenEditorsWidget(widget, () => !!this.editorWidgets.length),
+            isVisible: widget => this.withOpenEditorsWidget(widget, () => !!this.editorWidgets.length)
+        });
+        registry.registerCommand(OpenEditorsCommands.SAVE_ALL_TABS_FROM_TOOLBAR, {
+            execute: widget => this.withOpenEditorsWidget(widget, () => this.commandService.executeCommand(CommonCommands.SAVE_ALL.id)),
+            isEnabled: widget => this.withOpenEditorsWidget(widget, () => !!this.editorWidgets.length),
+            isVisible: widget => this.withOpenEditorsWidget(widget, () => !!this.editorWidgets.length)
+        });
+        registry.registerCommand(OpenEditorsCommands.CLOSE_ALL_IN_GROUP, {
+            execute: async (id): Promise<void> => {
+                const openEditorsWidget = await this.widgetManager.getOrCreateWidget<OpenEditorsWidget>(OpenEditorsWidget.ID);
+                const widgets = openEditorsWidget.getEditorWidgetsByGroup(id);
+                widgets?.forEach(widget => widget.close());
+            },
+            isVisible: () => false
```

Comment:
To confirm, the following is set to false to hide it from the `command palette`?

Reply:
So these two commands I've decided to hide from the command palette since they are essentially toolbar icon commands. Their implementation is slightly different from the way VSCode would implement them as they accept either an `ApplicationShell.Area` or `TabBar<Widget>` command as their argument.

Reply:
I thought a simpler way would be to not give it a `label` so it does not show up in the command palette?
Although, I'm fine with you explicitly setting it to `false` as well.